modules/virt-configuring-cluster-dpdk.adoc

```diff
@@ -63,24 +61,46 @@ metadata:
   name: profile-1
 spec:
   cpu:
-    isolated: 20,22,24,26,28,30,32,34,36,38,60,62,64,66,68,70,72,74,76,78
-    reserved: 0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,21,23,25,27,29,31,33,35,37,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,61,63,65,67,69,71,73,75,77,79
+    isolated: 4-39,44-79
+    reserved: 0-3,40-43
   hugepages:
     defaultHugepagesSize: 1G
     pages:
-      - count: 32
-        size: 1G
+    - count: 32
+      node: 0
+      size: 1G
+  net:
+    userLevelNetworking: true
+  nodeSelector:
+    node-role.kubernetes.io/worker-dpdk: ""
   numa:
     topologyPolicy: single-numa-node
-  nodeSelector:
-    node-role.kubernetes.io/worker-dpdk: "" 
 ----
 +
 [NOTE]
 ====
 The compute nodes automatically restart after you apply the `MachineConfigPool` and `PerformanceProfile` manifests.
 ====
 
+. Get the generated runtimeClass from the PerformanceProfile's status:
```

Comment:
```suggestion
. Retrieve the name of the generated `RuntimeClass` resource from the `status.runtimeClass` field of the `PerformanceProfile` object :
```

Reply:
This seems like a duplicate of the previous comment.